sys/tasklet/tasklet.c

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
This is not displayed as a rename by github. I didn't check if this is GitHubs fault, or if somehow in the commit this was not correctly detected as a rename.

Can you check if this is indeed not an issue with the commit, but only a bug in GitHub?